src/System.Reflection.TypeExtensions/ref/System.Reflection.TypeExtensions.cs

```diff
@@ -5,82 +5,82 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
-
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Reflection.BindingFlags))]
+
 namespace System.Reflection
 {
     public static partial class AssemblyExtensions
     {
-        public static System.Type[] GetExportedTypes(this System.Reflection.Assembly assembly) { throw null; }
-        public static System.Reflection.Module[] GetModules(this System.Reflection.Assembly assembly) { throw null; }
-        public static System.Type[] GetTypes(this System.Reflection.Assembly assembly) { throw null; }
+        public static System.Type[] GetExportedTypes(System.Reflection.Assembly assembly) { throw null; }
```

Comment:
Are these supposed to be extension methods? The source/ref disagree. Please review and let me know if this needs to be reverted. cc @steveharter, @GrabYourPitchforks

Reply:
Given that the type name is `AssemblyExtensions` and they were extension methods in 2.1 (https://github.com/dotnet/corefx/blob/release/2.1/src/System.Reflection.TypeExtensions/ref/System.Reflection.TypeExtensions.cs), the ref was right and source should be fixed.

Reply:
See the comment in the source. The discrepancy was deliberate. Declaring the impl as an extension method is jarring because it looks like infinite recursion, when in reality it's delegating to the instance method. But maybe it's better to change that to help with this generation. If so, fix the comment to say that the calls are not in fact recursive.

These API are now unfortunate warts acquired on reflection's long journey back to where it started. 

On all but some old TFMs, they won't be picked up at compile time because the actual instance methods that were brought back win.

They do have to remain for binary compat.